scripts/pre_commit_linter.py

```diff
@@ -633,13 +634,41 @@ def _check_bad_patterns(all_files):
     return summary_messages
 
 
+def _check_import_order(all_files):
+    """This function is used to check that each file
+    has imports placed in alphabetical order.
+    """
+    print 'Starting import-order checks'
+    print '----------------------------------------'
+    summary_messages = []
+    all_files = [
+        filename for filename in all_files if not
+        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)]
+    failed = False
+    for filename in all_files:
+        if isort.SortImports(filename, check=True).incorrectly_sorted:
```

Comment:
Just to confirm: does this line update the files with wrong import order as well, or does it just return True/False and do nothing else?

Reply:
This line prints the actual error message with the file path and returns True if it finds an error else False.

Reply:
Ah, that seems very side-effecty. Please add a comment above this line explaining that, since it's rather unexpected.